src/pages/home/dialogs/NewRecordsDialog.vue

```
@@ -0,0 +1,86 @@
+  message: {
+    type: String,
+    default: "Your class record is now ready to edit."
+  }
+});
+
+const emit = defineEmits(['close', 'update:show']);
+const router = useRouter();
+
+const navigateToRecentRecords = () => {
+  // Get all relevant data from localStorage
+  const classRecordId = localStorage.getItem("addedClassrecord");
```

ESLint

[error] 'classRecordId' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)